src/components/templates/PartyMap/components/Map/MapRoom.scss

```diff
@@ -12,9 +12,17 @@ $label-visible: flex;
   flex-direction: column;
   align-items: center;
 
+  appearance: none;
+  -webkit-appearance: none;
+  background: none;
+  border: none;
+  color: white;
+
   transition: transform 600ms $transition-function;
 
-  &:hover {
+  &:hover,
+  &:focus {
+    outline: none;
```

Comment:
Pretty sure removing `outline` like this creates an a11y issue, not resolves it.

Reply:
Please do NOT remove _outline_ anywhere. It's not good for accessibility.

Reply:
In this case item zooms when it is in focus. I back all outlines across the project. But in my opinion that it is not right. iOS accessibility mode draws system-level outlines. Default outlines only useful by navigating using "tab" button.

Reply:
‘Navigating with tab button’ is literally the point of accessibility..

Reply:
100% with @0xdevalias.  I repeat -- please do NOT "back all outlines across the project" @oleksii-front